### ISSUE TITLE
Compiled file Microsoft.Exchange.WebServices.NETStandard.dll available?

### ISSUE THREAD

**Issue by GruberMarkus** [ststeiger/RedmineMailService#1]
Hi @ststeiger,

I did not manage to compile Microsoft.Exchange.WebServices.NETStandard.dll successfully.

Could you please a compiled version of your source code?

Cheers,

GruberMarkus

**Comment by ststeiger** [comment]
Here you go: 
https://github.com/ststeiger/RedmineMailService/releases/download/2021-11-25/bin.zip
in
https://github.com/ststeiger/RedmineMailService/releases/tag/2021-11-25

**Comment by ststeiger** [comment]
What kind of error do you get ? 
Visual Studio version < 2019 ? 
2022 is latest.

**Comment by GruberMarkus** [comment]
Thank you very much!


Usually I only script and never have to compile anything.

I tried to compile it with a fresh install of Visual Studio 2022 Community Edition. There were quite a lot of warnings and some errors, especially around not finding commands or methods in the system.http namespace (the references were set in Visual Studio).
I decided to uninstall Visual Studio and to ask you, if you as developer could provide the already compiled dll.


While writing this, your dll passed all my test cases.
May I use it in my project Set-OutlookSignatures (https://github.com/GruberMarkus/Set-OutlookSignatures)?
There would of course be a referral to https://github.com/ststeiger/RedmineMailService and a copy of your license.txt.
